src/raw/client.rs

```diff
@@ -46,16 +47,17 @@ impl Client {
     /// # use tikv_client::{Config, RawClient};
     /// # use futures::prelude::*;
     /// # futures::executor::block_on(async {
-    /// let client = RawClient::new(Config::default()).await.unwrap().with_cf("write");
+    /// let client = RawClient::new(Config::default()).await.unwrap().with_cf("write").unwrap();
     /// let get_request = client.get("foo".to_owned());
     /// # });
     /// ```
-    pub fn with_cf(&self, cf: impl Into<ColumnFamily>) -> Client {
-        Client {
+    pub fn with_cf(&self, cf: &str) -> Result<Client> {
```

Comment:
Lets change `with_cf` to take `ColumnFamily` and let the caller handle the conversion. The example will then be:
` let client = RawClient::new(Config::default()).await.unwrap().with_cf(ColumnFamily::Write);`